docs/wiki/introduction/sql.md

```diff
@@ -499,6 +528,87 @@ There are also encoding functions available, to process query results.
     </p>
     </details>
 
+#### Collations
+
+- `version`:
+
+    <details>
+    <summary>Version collation example:</summary>
+    <p>
+
+      osquery> .mode line
+
+      osquery> select '1.0' = '1.0' collate version;
+      '1.0' = '1.0' collate version = 1
+
+      osquery> select '50.4.1b' < '50.4.1c' collate version;
+      '50.4.1b' < '50.4.1c' collate version = 1
+
+      osquery> select '20.10a' > '20.102' collate version;
+      '20.10a' > '20.102' collate version = 1
```

Comment:
```suggestion
      osquery> select '1.0' = '1.0' COLLATE VERSION;
      '1.0' = '1.0' COLLATE VERSION = 1

      osquery> select '50.4.1b' < '50.4.1c' COLLATE VERSION;
      '50.4.1b' < '50.4.1c' COLLATE VERSION = 1

      osquery> select '20.10a' > '20.102' COLLATE VERSION;
      '20.10a' > '20.102' COLLATE VERSION = 1
```

slight preference for SQL words in all caps

Reply:
Made those changes.